### PR TITLE
Relaxing the requirement of "iss" on jwt token to create claims identity

### PR DESCRIPTION
**Pull request by tushargupta51**
Fixing #154 #155 
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/182%23issuecomment-112225953%22%2C%20%22https%3A//github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/182%23issuecomment-112619599%22%2C%20%22https%3A//github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/182%23discussion_r32593329%22%2C%20%22https%3A//github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/182%23discussion_r32656032%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/182%23issuecomment-112225953%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%40brentschmaltz%20%22%2C%20%22created_at%22%3A%20%222015-06-15T22%3A15%3A28Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/803796%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tushargupta51%22%7D%7D%2C%20%7B%22body%22%3A%20%22Added%20ValidateSignature%20boolean%20flag%20on%20TVP.%22%2C%20%22created_at%22%3A%20%222015-06-17T01%3A30%3A41Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/803796%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tushargupta51%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%203be7615318bc259c574cdb001a8f1c200b89570f%20src/System.IdentityModel.Tokens/Tokens/TokenValidationParameters.cs%206%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/182%23discussion_r32593329%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22This%20attribute%20doesn%27t%20set%20the%20value%20to%20true.%5Cr%5CnTest%3F%22%2C%20%22created_at%22%3A%20%222015-06-17T03%3A43%3A11Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3172421%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/brentschmaltz%22%7D%7D%2C%20%7B%22body%22%3A%20%22I%20understood%20it%27s%20usage%20incorrectly.%20Added%20tests%20for%20both%20positive%20and%20negative%20cases.%22%2C%20%22created_at%22%3A%20%222015-06-17T18%3A05%3A47Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/803796%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tushargupta51%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20src/System.IdentityModel.Tokens/Tokens/TokenValidationParameters.cs%3AL471-487%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/182#issuecomment-112225953'>General Comment</a></b>
- <a href='https://github.com/tushargupta51'></a> @brentschmaltz
- <a href='https://github.com/tushargupta51'></a> Added ValidateSignature boolean flag on TVP.
- [ ] <a href='#crh-comment-Pull 3be7615318bc259c574cdb001a8f1c200b89570f src/System.IdentityModel.Tokens/Tokens/TokenValidationParameters.cs 6'></a> &nbsp;<b><a href='https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/182#discussion_r32593329'>File: src/System.IdentityModel.Tokens/Tokens/TokenValidationParameters.cs:L471-487</a></b>
- <a href='https://github.com/brentschmaltz'></a> This attribute doesn't set the value to true.
Test?
- <a href='https://github.com/tushargupta51'></a> I understood it's usage incorrectly. Added tests for both positive and negative cases.

<a href='https://www.codereviewhub.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/182?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/182?approve=1'></a>&nbsp;<a href='https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/182'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by brandwe** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/182'>Refresh</a>.<a href='#crh-update-none'></a>

**Comment by tushargupta51** [comment]
@brentschmaltz

**Comment by tushargupta51** [comment]
Added ValidateSignature boolean flag on TVP.

File: src/System.IdentityModel.Tokens/Tokens/TokenValidationParameters.cs [hunk 1]
```diff
@@ -471,6 +471,16 @@ public bool ValidateIssuerSigningKey
         }
 
         /// <summary>
+        /// Gets or sets the boolean to control if the signature of the token will be validated during token validation.
+        /// </summary>
+        [DefaultValue(true)]
```
Comment (brentschmaltz) [inline src/System.IdentityModel.Tokens/Tokens/TokenValidationParameters.cs]: This attribute doesn't set the value to true.
Test?

Comment (tushargupta51) [inline src/System.IdentityModel.Tokens/Tokens/TokenValidationParameters.cs]: I understood it's usage incorrectly. Added tests for both positive and negative cases.

**Pull request merged**